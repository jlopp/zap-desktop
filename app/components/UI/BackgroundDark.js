import React from 'react'
import { Box } from 'rebass'

/**
 * @render react
 * @name BackgroundDark
 * @example
 * <BackgroundDark />
 */
class BackgroundDark extends React.Component {
  render() {
    return <Box bg="darkestBackground" color="white" {...this.props} />
  }
}

export default BackgroundDark
