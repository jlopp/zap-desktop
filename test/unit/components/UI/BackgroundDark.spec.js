import React from 'react'
import BackgroundDark from 'components/UI/BackgroundDark'
import renderer from 'react-test-renderer'

describe('component.UI.BackgroundDark', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<BackgroundDark />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
