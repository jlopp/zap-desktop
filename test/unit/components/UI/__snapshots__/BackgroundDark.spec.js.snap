// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component.UI.BackgroundDark should render correctly 1`] = `
.c0 {
  color: white;
  background-color: darkestBackground;
}

<div
  className="c0"
  color="white"
/>
`;
